powsybl-geo-data-server/src/main/java/com/powsybl/data/store/server/DataStoreServer.java

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.data.store.server;
+
+import com.powsybl.data.store.server.repositories.*;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.network.store.client.NetworkStoreService;
+import infrastructure.LineGraphic;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import com.powsybl.data.store.server.repositories.LinesCustomRepository;
+import server.utils.NetworkGeoData;
+import infrastructure.SubstationGraphic;
+import io.swagger.annotations.Api;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+@RestController
+@RequestMapping(value = DataStoreServer.API_VERSION)
+@Api(value = "/geo/data/store", tags = "rte Geo data store")
+@ComponentScan(basePackageClasses = {DataStoreServer.class, NetworkGeoData.class, NetworkStoreService.class})
+public class DataStoreServer {
+
+    static final String API_VERSION = "v1";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DataStoreServer.class);
+
+    @Autowired
+    private NetworkGeoData networkGeoData;
+
+    @Autowired
+    private NetworkStoreService service;
+
+    @Autowired
+    private SubstationsRepository substationsRepository;
+
+    @Autowired
+    private LinesRepository linesRepository;
+
+    @Autowired
+    private LinesCustomRepository linesCustomRepository;
+
+    @GetMapping(value = "lines-graphics-with-pagination/{idNetwork}")
```

Comment:
I am wondering if it is  a good design to have an endpoint with (*-pagination). Maybe it could be better to add an extra query parameter to an existing endpoint for that like
lines-graphics?pagination=true